build.go

```diff
@@ -98,13 +137,12 @@ func Build(
 		logger.Break()
 
 		assetsLayer.Launch = true
+		assetsLayer.LaunchEnv.Default("RAILS_ENV", "production")
```

Comment:
The values for `RAILS_ENV` and `RAILS_SERVE_STATIC_FILES` should be configurable. Maybe using a environment variable passed to `pack`.

Reply:
Made it

Reply:
Made what? Please describe what you did so I can review it.

Reply:
assetsLayer.Launch = true
		if val, ok := os.LookupEnv("RAILS_ENV"); ok {
			assetsLayer.LaunchEnv.Default("RAILS_ENV", val)
		} else {
			assetsLayer.LaunchEnv.Default("RAILS_ENV", "production")
			os.Setenv("RAILS_ENV", "production")
		}
		if val, ok := os.LookupEnv("RAILS_SERVE_STATIC_FILES"); ok {
			assetsLayer.LaunchEnv.Default("RAILS_SERVE_STATIC_FILES", val)
		} else {
			assetsLayer.LaunchEnv.Default("RAILS_SERVE_STATIC_FILES", "true")
			os.Setenv("RAILS_SERVE_STATIC_FILES", "true")
		}